pkgs/by-name/jm/jmeter/package.nix

```diff
@@ -20,38 +21,167 @@ stdenv.mkDerivation rec {
     jre
   ];
 
+  # Wrapper script to handle read-only Nix store files for JMeter report generation
```

Comment:
I don't like this very complex wrapper. Can't we patch the code instead? Can this be made simpler and half as long in the process?

Reply:
Yea. I am going to move the package to that so we can apply patches.